src/styles/blocks/product.scss

```diff
@@ -0,0 +1,136 @@
+.catalog {
+  display: flex;
+  justify-content: center;
+}
+
+.product {
+  display: flex;
+  width: 944px;
```

Comment:
use scss variables to determine the width of the container (it consists of card width and the spacing between each of them), don't hardcode them in px